packages/sandbox/src/app/sandbox/docs/button.doc.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Button } from '@getluko/streamline';
+
+export const ButtonSandbox = () => (
+  <Box paddingHorizontal="xl">
+    <Box paddingVertical="sm">
+      <Button variant="primary" text="hello world" />
+    </Box>
+    <Box paddingVertical="sm">
+      <Button variant="primary" isDisabled text="hello world" />
+    </Box>
+    <Box paddingVertical="sm">
+      <Button variant="secondary" text="hello world" />
+    </Box>
+    <Box paddingVertical="sm">
+      <Button variant="neutral" text="hello world" />
+    </Box>
+    <Box paddingVertical="sm">
+      <Button variant="danger" text="hello world" />
+    </Box>
+    <Box paddingVertical="sm">
+      <Button variant="placeholder" text="hello world" />
+    </Box>
+  </Box>
+);
```

Comment:
update cases with Icons, size:mini, loading
generate screenshots on iOS and Android